src/styles/globals.scss

```
@@ -0,0 +1,46 @@
+@import "tailwindcss";
+
+// ---------- 컬러 색상 ---------- //
+@theme {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)